Tests/BUILD

```diff
@@ -13,17 +13,17 @@ genrule(
     name = "LinuxMain",
     srcs = ["LinuxMain.swift"],
     outs = ["main.swift"],
-    cmd = "cp $< $@"
+    cmd = "cp $< $@",
 )
 
 swift_test(
-    name = "YamsTests",
-    module_name = "YamsTests",
+    name = "UnitTests",
```

Comment:
Can we keep this named YamsTests?

Reply:
Checkout the updated description

Reply:
Ah yes that clears it up.

Reply:
found https://github.com/bazelbuild/bazel/issues/12312 for future reference